spec/logman_spec.rb

```diff
@@ -19,6 +19,13 @@
     expect(Logman::VERSION).not_to be nil
   end
 
+  it "behaves like standard ruby logger" do
+    logger = Logger.new(STDOUT)
+    Logman.new(:logger => logger)
+
+    logger.info("This is string log message")
```

Comment:
Maybe:

```
expect { logger.info("This is string log message") }.to .to output("This is string log message").to_stdout
```

http://rspec.info/blog/2014/05/notable-changes-in-rspec-3/#new-output-matcher